README.md

```diff
@@ -101,6 +101,33 @@ module.exports = {
 };
 ```
 
+### errorOptions
+
+Allows to set various combinations of error notifications inside xml.
```

Comment:
Can we add a little bit more description for it? I personally wouldn't know for what this is for.